kubernetes_env/kube_env.py

```diff
@@ -54,10 +55,10 @@ def deploy_addons():
     apply_cmd = "kubectl apply -f "
     url = "https://raw.githubusercontent.com/istio/istio/release-1.9"
     cmd = f"{apply_cmd} {YAML_DIR}/prometheus-mod.yaml && "
-    cmd += f"{apply_cmd} {url}/samples/addons/jaeger.yaml && "
-    cmd += f"{apply_cmd} {url}/samples/addons/grafana.yaml "
-    # cmd += f"{apply_cmd} {url}/samples/addons/kiali.yaml || "
-    # cmd += f"{apply_cmd} {url}/samples/addons/kiali.yaml"
+    # Kiali needs to run twice from this issue: github.com/istio/istio/issues/27417
+    addons = ["jaeger", "grafana", "kiali", "kiali"]
+    cmd += " && ".join(
+        [f"{apply_cmd} {url}/samples/addons/{addon}.yaml" for addon in addons])
```

Comment:
How does the input work? This is ideally a list right? I think this needs to change a little. Prometheus is also an addon.

Reply:
I changed it to letting user inputting addons name instead.